src/postgres/src/test/regress/expected/yb_pg_join.out

```diff
@@ -0,0 +1,6598 @@
+     | 2 | 3 | two   |  2
+     | 2 | 3 | two   |  4
+     | 3 | 2 | three | -3
+     | 4 | 1 | four  |
+     | 5 | 0 | five  | -5
+     | 5 | 0 | five  | -5
+     | 6 | 6 | six   |
+     | 7 | 7 | seven |
+     | 8 | 8 | eight |
+     |   |   | null  |
+     |   | 0 | zero  |
+(13 rows)
+
+SELECT '' AS "xxx", *
+  FROM J1_TBL LEFT JOIN J2_TBL USING (i)
+  ORDER BY i, k, t;
+ xxx | i | j |   t   | k
+-----+---+---+-------+----
+     | 0 |   | zero  |
+     | 1 | 4 | one   | -1
+     | 2 | 3 | two   |  2
+     | 2 | 3 | two   |  4
+     | 3 | 2 | three | -3
+     | 4 | 1 | four  |
+     | 5 | 0 | five  | -5
+     | 5 | 0 | five  | -5
+     | 6 | 6 | six   |
+     | 7 | 7 | seven |
+     | 8 | 8 | eight |
+     |   |   | null  |
+     |   | 0 | zero  |
+(13 rows)
+
+SELECT '' AS "xxx", *
+  FROM J1_TBL RIGHT OUTER JOIN J2_TBL USING (i) order by 1, 2, 3, 4, 5;
+ xxx | i | j |   t   | k
+-----+---+---+-------+----
+     | 0 |   | zero  |
+     | 1 | 4 | one   | -1
+     | 2 | 3 | two   |  2
+     | 2 | 3 | two   |  4
+     | 3 | 2 | three | -3
+     | 5 | 0 | five  | -5
+     | 5 | 0 | five  | -5
+     |   |   |       |  0
+     |   |   |       |
+(9 rows)
+
+SELECT '' AS "xxx", *
+  FROM J1_TBL RIGHT JOIN J2_TBL USING (i) order by 1, 2, 3, 4, 5;
+ xxx | i | j |   t   | k
+-----+---+---+-------+----
+     | 0 |   | zero  |
+     | 1 | 4 | one   | -1
+     | 2 | 3 | two   |  2
+     | 2 | 3 | two   |  4
+     | 3 | 2 | three | -3
+     | 5 | 0 | five  | -5
+     | 5 | 0 | five  | -5
+     |   |   |       |  0
+     |   |   |       |
+(9 rows)
+
+SELECT '' AS "xxx", *
+  FROM J1_TBL FULL OUTER JOIN J2_TBL USING (i)
+  ORDER BY i, k, t;
+ xxx | i | j |   t   | k
+-----+---+---+-------+----
+     | 0 |   | zero  |
+     | 1 | 4 | one   | -1
+     | 2 | 3 | two   |  2
+     | 2 | 3 | two   |  4
+     | 3 | 2 | three | -3
+     | 4 | 1 | four  |
+     | 5 | 0 | five  | -5
+     | 5 | 0 | five  | -5
+     | 6 | 6 | six   |
+     | 7 | 7 | seven |
+     | 8 | 8 | eight |
+     |   |   |       |  0
+     |   |   | null  |
+     |   | 0 | zero  |
+     |   |   |       |
+(15 rows)
+
+SELECT '' AS "xxx", *
+  FROM J1_TBL FULL JOIN J2_TBL USING (i)
+  ORDER BY i, k, t;
+ xxx | i | j |   t   | k
+-----+---+---+-------+----
+     | 0 |   | zero  |
+     | 1 | 4 | one   | -1
+     | 2 | 3 | two   |  2
+     | 2 | 3 | two   |  4
+     | 3 | 2 | three | -3
+     | 4 | 1 | four  |
+     | 5 | 0 | five  | -5
+     | 5 | 0 | five  | -5
+     | 6 | 6 | six   |
+     | 7 | 7 | seven |
+     | 8 | 8 | eight |
+     |   |   |       |  0
+     |   |   | null  |
+     |   | 0 | zero  |
+     |   |   |       |
+(15 rows)
+
+SELECT '' AS "xxx", *
+  FROM J1_TBL LEFT JOIN J2_TBL USING (i) WHERE (k = 1) order by 1, 2, 3, 4, 5;
+ xxx | i | j | t | k
+-----+---+---+---+---
+(0 rows)
+
+SELECT '' AS "xxx", *
+  FROM J1_TBL LEFT JOIN J2_TBL USING (i) WHERE (i = 1) order by 1, 2, 3, 4, 5;
+ xxx | i | j |  t  | k
+-----+---+---+-----+----
+     | 1 | 4 | one | -1
+(1 row)
+
+--
```

Comment:
Explain joins start to get different from here.  I think we can either trim the file at this point or keep the differing join results.  I lean towards keeping so that we can see when we improve the join cost estimates.